### ISSUE TITLE
Accessibility - All Incidents Screen - Not Showing Any Incidents

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#36]
Since the application is being used by other people when I tapped on the “All Incidents” link/tab, I don’t see any other issues. I would have expected to see some of the other people's information or at least my own testing information and not having a blank main content section of the screen. If it is empty, then there should be a message telling the user there is no incident in the system.

There is a similar issue for the "My Incidents" screen, and the only thing that changed and wasn’t announced to the VoiceOver user is the heading “All Reported Incidents” and the "My Incidents" along with the same for the heading at the top of the screen.

**Comment by aanthonyDOL** [comment]
Duplicate of another issue. Had to add permissions to the user.
